.github/workflows/tpu_ci.yml

```diff
@@ -10,6 +10,8 @@ jobs:
     runs-on: v4-runner-set
     steps:
       - name: Checkout and Setup PyTorch Repo
+        env:
+          _GLIBCXX_USE_CXX11_ABI: 0
```

Comment:
The C++11 ABI switch has been such an unnecessary headache. I don't know why the default for building `torch` doesn't match their published wheels....